frontend/app/src/components/history/ledger-actions/LedgerActionDetails.vue

```diff
@@ -47,5 +74,31 @@ defineProps({
   }
 });
 
+const { item } = toRefs(props);
+const { dark } = useTheme();
+
+const hasLink = computed(() => {
+  const { link } = get(item);
+  return link && link.startsWith('http');
+});
+
 const { t } = useI18n();
+const { isPackaged, openUrl } = useInterop();
+
+const target = isPackaged ? undefined : '_blank';
+
+const href = computed(() => {
```

Comment:
Do you think we could use the link composable here? it is in `composables/link.ts` if I am not mistaken